CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ### Changed
 - The templates are now based on Bootstrap 4.
+- `DisableView` now properly checks user has verified before disabling
+  two-factor on their account
 
 ## 1.12 - 2020-07-08
```

Comment:
I’d remove “properly” here. We don’t support any other login flows than our own, thus 2fa was already verified on sign-in. From the docs:

> Warning:  
> Be sure to remove any other login routes, otherwise the two-factor authentication might be circumvented.

Reply:
Done :+1: